build.gradle

```diff
@@ -9,11 +9,13 @@ buildscript {
     dependencies {
```

Comment:
Please remove this file from the commit as there are no changes to it related to the PR